test/CMakeLists.txt

```diff
@@ -113,6 +113,31 @@ function(get_test_dependencies SDK result_var_name)
   set("${result_var_name}" "${deps}" PARENT_SCOPE)
 endfunction()
 
+function (write_component_test_script test_bin_dir component_name)
+  string(REPLACE ";" " " args "${LIT_ARGS}")
+  file(
+    WRITE
+    ${test_bin_dir}/check-swift-${component_name}.sh
+    "${PYTHON_EXECUTABLE} ${LIT} \\
+    ${args} \\
+    ${test_bin_dir}/ \\
+    --filter=${component_name}\n")
+
+  file(
```

Comment:
Why are there install rules for these?  These really shouldn't be installed, they are internal developer things.

Reply:
To write the file with execute permissions.

Reply:
Then make this hack more egregious: use a post command to `chmod +x` it.  Alternatively, you could write the scripts up front with the proper permissions and use `configure_file` to replace the output location and python executable paths.

Reply:
I ended up using a template script and `configure_file`, thanks for the suggestion!